src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return 80;
+  }
+
+  if (days === 3) {
+    return 100;
+  }
+
+  if (days === 6) {
+    return 220;
+  }
+
+  if (days >= 7) {
+    return 230;
```

Comment:
The return value for 7 or more days is hard-coded to 230, which doesn't scale with the number of days beyond 7. Implement a dynamic calculation using the daily rate and the appropriate long-term discount.